### PR TITLE
prompt processing: add production environment

### PR DESCRIPTION
**Pull request by dspeck1**
Add production environment and enable prompt proto service for latiss

### CODE REVIEW

File: charts/prompt-proto-service/values.yaml [hunk 1]
```diff
@@ -42,6 +42,9 @@ registry:
   ip: ""
   db: ""
   user: ""
+  centralRepoFile: false
+
+logLevel: lsst.resources=DEBUG
```
Comment (kfindeisen) [inline charts/prompt-proto-service/values.yaml:47]: In my experience, `lsst.resources` logs are _extremely_ obnoxious, and turning them on by default (for both `-dev` and `-prod`, and all instruments?) will make it impossible to discern anything we might be interested in at the `DEBUG` level (including our own diagnostics).

I see that this was copied from https://github.com/slaclab/rubin-usdf-prompt-processing/blob/main/kubernetes/overlays/prod/prompt-proto-service/prompt-proto-service.yaml#L102, but would it be possible to narrow the scope a bit?

Comment (dspeck1) [reply charts/prompt-proto-service/values.yaml:47]: What default would you like?

Comment (kfindeisen) [reply charts/prompt-proto-service/values.yaml:47]: If it's not possible to leave `SERVICE_LOG_LEVELS` undefined, then `""` would have the same effect.

Comment (dspeck1) [reply charts/prompt-proto-service/values.yaml:47]: Thanks.  Will set default to `""` in separate PR.

**Pull request merged**